src/platform/forms/tests/save-in-progress/SaveInProgressIntro.unit.spec.jsx

```diff
@@ -50,6 +53,10 @@ describe('Schemaform <SaveInProgressIntro>', () => {
       />,
     );
 
+    expect(
+      tree.find('.saved-form-item-metadata').get(1).props.children[1],
+    ).to.equal(moment.unix(946684800).format('M/D/YYYY [at] h:mm a'));
```

Comment:
I'd love to hear your thoughts on this, but my instinct would be to hard code the formatted date we are expecting here. In this case it's _highly_ unlikely that Moment will get wonky in a future version, but if it did this test wouldn't catch it.

Reply:
I don't know what timezone to code it for. what I code for in Utah will probably fail on servers in Virginia.

Reply:
heh, totally valid point!

Reply:
I think this implementation could be improved if we didn't have components creating data. would be easier to test if it was just a pure function whose value was injected through a prop.